modules/acquia_cms_headless/src/Plugin/Field/FieldFormatter/OEmbedFormatter.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+namespace Drupal\acquia_cms_headless\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\Url;
+use Drupal\media\Entity\MediaType;
+use Drupal\media\IFrameUrlHelper;
+use Drupal\media\OEmbed\Resource;
+use Drupal\media\OEmbed\ResourceException;
+use Drupal\media\OEmbed\ResourceFetcherInterface;
+use Drupal\media\OEmbed\UrlResolverInterface;
+use Drupal\media\Plugin\media\Source\OEmbedInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+// cspell:ignore allowtransparency
+/**
+ * Plugin implementation of the 'oembed' formatter.
+ *
+ * @internal
+ *   This code si copied from drupal core oEmbed.
```

Comment:
@chandan-singh7929 Minor spelling correction.
```suggestion
 *   This code is copied from drupal core oEmbed.
```

Reply:
Done